### PR TITLE
ClusterLoader - Logging profile error for gke

### PR DESCRIPTION
**Pull request by krzysied**
Profiles will only log master sshing error instead of propagating it to caller.
Gathering profiles from apiserver is not a standard thing in gke.

### CODE REVIEW

File: clusterloader2/pkg/measurement/common/simple/profile.go [hunk 1]
```diff
@@ -102,6 +103,11 @@ func gatherProfile(componentName, profileKind, host, provider string) ([]measure
 	getCommand := fmt.Sprintf("curl -s localhost:%v/debug/pprof/%s", profilePort, profileKind)
 	sshResult, err := measurementutil.SSH(getCommand, host+":22", provider)
 	if err != nil {
+		if provider == "gke" {
+			// Only logging error for gke. SHHing to gke master is not a standard case.
```
Comment (wojtek-t) [inline clusterloader2/pkg/measurement/common/simple/profile.go]: s/is not a standard case/is not supported/

Comment (krzysied) [reply clusterloader2/pkg/measurement/common/simple/profile.go]: Done.

**Pull request merged**